tests/admin.test.ts

```diff
@@ -182,3 +185,195 @@ describe('bonus points submission', () => {
     expect(getNoBonusUserResponse.user.points).toEqual(0);
   });
 });
+
+describe('updating user access level', () => {
+  test('updates the access level of the user', async () => {
+    const conn = await DatabaseConnection.get();
+    const admin = UserFactory.fake({ accessType: UserAccessType.ADMIN });
+    const users = UserFactory.create(4);
+
+    users[0].accessType = UserAccessType.STAFF;
+    users[1].accessType = UserAccessType.STANDARD;
+    users[2].accessType = UserAccessType.MARKETING;
+    users[3].accessType = UserAccessType.MERCH_STORE_DISTRIBUTOR;
+
+    await new PortalState()
+      .createUsers(users[0], users[1], users[2], users[3], admin)
+      .write();
+
+    const adminController = ControllerFactory.admin(conn);
+
+    const accessLevelResponse = await adminController.updateUserAccessLevel({
+      accessUpdates: [
+        { user: users[0].email, accessType: UserAccessType.MERCH_STORE_MANAGER },
+        { user: users[1].email, accessType: UserAccessType.MARKETING },
+        { user: users[2].email, accessType: UserAccessType.MERCH_STORE_DISTRIBUTOR },
+        { user: users[3].email, accessType: UserAccessType.STAFF },
+      ],
+    }, admin);
+
+    const repository = conn.getRepository(UserModel);
+    const updatedUsers = await repository.find({
+      email: In([users[0].email, users[1].email, users[2].email, users[3].email]),
+    });
+
+    // expect statements
```

Comment:
nit: instead of saying "expect statement", let's add a comment right above "const repository..." line that says "verify that the changes correctly propagated to the database"